app/src/ereferrals/java/org/eyeseetea/malariacare/services/strategies/PushServiceStrategy.java

```diff
@@ -176,7 +175,7 @@ protected void executePush() {
                 new PushUseCase(pushController, asyncExecutor, mainExecutor,
                         surveysThresholds, surveyRepository, orgUnitRepository);
 
-        SurveyChecker.launchQuarantineChecker();
+        surveyRepository.updateAllQuarantineSurveysAsComplete();
```

Comment:
the decision of update quarantine surveys to complete is a domain decision, not a presentation decision then the Push use case should realize this decision.